ml_inference_offloading/src/test/java/ai/nnstreamer/ml/inference/offloading/ActivityUnitTest.kt

```diff
@@ -0,0 +1,25 @@
+package ai.nnstreamer.ml.inference.offloading
+
+
+import androidx.recyclerview.widget.RecyclerView
+
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.Robolectric
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class ActivityUnitTest {
+    @Test
+    fun testModelListLength() {
+        val activityController = Robolectric.buildActivity(MainActivity::class.java).create()
+        val activity = activityController.get()
+        val recyclerview = activity.findViewById<RecyclerView>(R.id.model_list)
+        val adapter = recyclerview.adapter
+        assertNotNull(adapter)
+        val itemCount = adapter!!.itemCount
```

Comment:
```adapter``` is not null at this line.
```suggestion
adapter?.run {
      assertEquals(2, adapter.itermCount)
}
```

I am not sure this code is working:)

Reply:
I also thought `adapter` null handling is something wierd and added `assertNotNull(adpater)`.
It looks much clear to use `run`
Thank you :)